### DISCUSSION TITLE
I can't change wifi setting

### DISCUSSION THREAD

**Discussion by nicecube** [Blackymas/NSPanel_HA_Blueprint#1921] [General]
I migrated my HA instance to another VLAN/Subnet/Wi-Fi network. Now, I want to change the Wi-Fi configuration of my NSPanel, but I'm encountering errors when I try to update the information in ESPHome. Can someone assist me?

I tried adding a static IP, but I'm unsure where to input it, as I don't see the Wi-Fi section like in other ESPHome instances.

Here's the error I'm getting:

INFO Successfully compiled program.
INFO Connecting to 172.16.82.247
INFO Uploading /data/build/nswhite01/.pioenvs/nswhite01/firmware.bin (1129760 bytes)
ERROR Error auth result: Error: Authentication invalid. Is the password correct?

The new subnet is 172.16.80.x.

**Comment by edwardtfn** [comment]
Please take a look at these:

- [How to - Updating Wi-Fi settings and OTA Passwords in ESPHome with This Project](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/howto.md#updating-wi-fi-settings-and-ota-passwords-in-esphome-with-this-project)
- #1894
- #1121
